notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOMTN-092_inertia_compensation_single_slew_hp.ipynb

```diff
@@ -79,6 +79,7 @@
    "outputs": [],
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #37.        max_ax.axvline(450, ls=":", c="red", alpha=0.5, label="Operational limit")
</pre><p>A very minor suggestion: maybe with another color for one of that? (like orange or fainter red color for operation limit, etc). </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/136/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>